static/docs/use-cases/huge-data-on-an-external-drive.md

```diff
@@ -0,0 +1,156 @@
+# Huge Data On An External Local Drive
+
+Sometimes the data may be huge and they are stored on an external local drive.
+By "huge" we mean that they won't fit on our home directory, and even if they
+did, it would certainly take a long time to copy them back an forth from the
+external drive to our home directory. For example let's say that the data are
+stored on an external HDD drive of size 16TB, which is mounted on `/mnt/data/`,
+while the disk of our home directory has a size of only 320GB.
+
+In this case we would like to process the data where they are (on the external
```

Comment:
"the data where they are" -> "the data where it's already located"

Reply:
Repeated in https://github.com/iterative/dvc.org/pull/565#pullrequestreview-276956619